docker/src/main/java/brooklyn/entity/container/docker/application/VanillaDockerApplication.java

```diff
@@ -54,6 +54,9 @@
     @SetFromFlag("imageTag")
     ConfigKey<String> IMAGE_TAG = DockerAttributes.DOCKER_IMAGE_TAG.getConfigKey();
 
+    @SetFromFlag("entrypoint")
+    ConfigKey<List<String>> IMAGE_ENTRYPOINT = DockerAttributes.DOCKER_IMAGE_ENTRYPOINT.getConfigKey();
```

Comment:
I am trying to be consistent with the other configuration keys here, hence the aliasing of the field name